docs/program/sdks/_index.md

```diff
@@ -157,52 +157,31 @@ The TypeScript SDK currently only supports building web browser apps.
 {{< /alert >}}
 
 ```ts {class="line-numbers linkable-line-numbers"}
-import { Client, createRobotClient, RobotClient } from '@viamrobotics/sdk';
+// This code must be run in a browser environment.
 
-async function connect() {
-  // You can remove this block entirely if your robot is not authenticated.
-  // Otherwise, replace with an actual secret.
-  const secret = '<SECRET>';
-  const credential = {
-    payload: secret,
-    type: 'robot-location-secret',
-  };
+import * as VIAM from '@viamrobotics/sdk';
 
-  // Replace with the host of your actual robot running Viam.
-  const host = "<HOST>";
-
-  // Replace with the signaling address. If you are running your robot on Viam,
-  // it is most likely https://app.viam.com:443.
-  const signalingAddress = 'https://app.viam.com:443';
-
-  const iceServers = [{ urls: 'stun:global.stun.twilio.com:3478' }];
+async function main() {
+  const host = 'rover-main-main.yriemen2i2.viam.cloud';
 
-  return createRobotClient({
+  const robot = await VIAM.createRobotClient({
     host,
-    credential,
+    credential: {
+      type: 'robot-location-secret',
+      payload: 'd7fxn4zc92cjwmlhd7zh545la0xmpc71nnn9gmtyb28g0nhd',
+    },
     authEntity: host,
-    signalingAddress,
-    iceServers
+    signalingAddress: 'https://app.viam.com:443',
   });
-}
+  
 
-async function main() {
-  // Connect to client
-  let client: Client;
-  try {
-    client = await connect();
-    console.log('connected!');
-
-    let resources = await client.resourceNames();
-    console.log('Resources:');
-    console.log(resources);
-  } catch (error) {
-    console.log(error);
-    return;
-  }
+  console.log('Resources:');
+  console.log(await robot.resourceNames());
 }
 
-main();
+main().catch((error) => {
+  console.error('encountered an error:', error)
+});
 ```
```

Comment:
Looks like we have largely changed this code sample since initial publish. This is the current code sample content shown today.